<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createMatchWindow</class>
 <widget class="QDialog" name="createMatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>378</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>match</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2F2F2F;</string>
  </property>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>220</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear partida</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputNameMatch">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameMatch">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre  partida:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaps">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seleccionar mapa:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listMaps">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>50</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget::item:selected {
    background-color: orange;  /* fondo azul */
    color: white;              /* texto blanco */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="waitLabel">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>320</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButon">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>220</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Atras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Comenzar Partida</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
